### PR TITLE
feat(exchangerates): add exchangerates pipeline

### PR DESCRIPTION
**Pull request by mike-bolshakov**
A pipeline fetching exchange rates data from https://exchangeratesapi.io/.
Service provides 250 API requests monthly for free for a lifetime.
Pipeline will import a set of currencies to be converted to one single base currency.

### CODE REVIEW

File: pipelines/exchangerates.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from exchange_rates import exchangerates_source
+
+CURRENCY_LIST = ["AUD", "BRL", "CAD", "CHF", "CNY", "DKK", "EUR", "GBP"]
+BASE_CURRENCY = "EUR"
+
+pipeline = dlt.pipeline(
+        pipeline_name="exchangerates", destination="postgres", dataset_name="exchangerates_data"
+    )
+
+def get_last_updated_at(default="2023-03-17T00:00:00Z"):
```
[the remark this answers is not in the document]
Comment (adrianbr) [reply pipelines/exchangerates.py:11]: the entire source functionality should be contained in the source folder. The pipeline file should only showcase how to call the source and pass any arguments

[the remark this answers is not in the document]
Comment (mike-bolshakov) [reply pipelines/exchangerates.py:11]: @adrianbr understood. I guess, i didnt quite realize (didnt read to the docs) that the state/bookmark gets committed and stored in db :(

**Comment by mike-bolshakov** [comment]
@rudolfix Hi, here is another pipeline of mine, which i believe/hope ready to be reviewed and checked.
If you could put it through a workflow, that would be great. thank you!

**Comment by burnash** [comment]
Hey @mike-bolshakov,

Thank you for your awesome PR! We're shaking things up and moving all community pipelines to another repo to keep things tidy. I'm going to close this PR but we'll keep the branch alive!

Your work is much appreciated, let me know if you've got any questions.

Cheers!

**Pull request closed without merging**